<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1571</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="add_object">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD OBJECT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="obj_list">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Cube</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cylinder</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sphere</string>
     </property>
    </item>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ADDED ITEMS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_object">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_x">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_y">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_z">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.250000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_x">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_y">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_z">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-360.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_x">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_y">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale_z">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>64</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-2.000000000000000</double>
    </property>
    <property name="maximum">
     <double>2.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>move</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rotate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TEXTURE</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>1281</width>
      <height>981</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>540</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_texture">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Projective</string>
    </property>
   </widget>
   <widget class="QPushButton" name="procedural">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Procedural</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bump">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>590</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bump</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>690</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>LIGHT SOURCE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="left_light">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="right_light">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>760</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="down_light">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>810</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="top_light">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>720</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="near_light">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>750</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Near</string>
    </property>
   </widget>
   <widget class="QPushButton" name="far_light">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>780</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Far</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1571</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
